### PR TITLE
Update main.c

### PR DESCRIPTION
**Pull request by Sandroid75**
added real-time elapsed time display
added elapsed time for player solution

### CODE REVIEW

File: src/.clang-format [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
```
Comment (jubalh) [inline src/.clang-format:1]: I don't want this file.

File: src/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (jubalh) [inline src/.vscode/launch.json:1]: I don't want this file.

File: src/.vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (jubalh) [inline src/.vscode/tasks.json:1]: I don't want this file.

File: src/main.c [hunk 4]
```diff
@@ -4,6 +4,7 @@ vim: noexpandtab:ts=4:sts=4:sw=4
 nudoku
 
 Copyright (C) 2014 - 2020 Michael "jubalh" Vetter - jubalh _a-t_ iodoru.org
+Copyright (C) 2022 Sandroid75 - real-time elapsed playing time
```
Comment (jubalh) [inline src/main.c]: just because one change?

File: src/main.c [hunk 5]
```diff
@@ -446,8 +453,24 @@ static bool hint(void)
 	return false;
 }
 
+/* function to compose message string with elapsed time */
+static void elapsed_time_str(const char *msg, char *time_string)
+{
+	int hours, minutes, seconds;	//variables to compose time string hh:mm:ss
+
+	if(g_playing)			//only if playing
```
Comment (jubalh) [inline src/main.c:460]: this comment is obvious. Generally there are too many comments. Just comment some details that are not easily understood by the code itself.

**Review by jubalh — CHANGES_REQUESTED** [review]
See my review comments.

`Update main.c` doesn't say anything.
You are adding a feature. You could just state this in the commit message. `Add timer feature` or something like this.

File: src/main.c (hunk 4 above)
Comment (Sandroid75) [reply src/main.c]: of course you can't delete this comment, it's just a small contribution.

File: src/main.c (hunk 5 above)
Comment (Sandroid75) [reply src/main.c:460]: I haven't been producing software for many years now, now it's just for hobby and fun, I like to comment to be useful to those who are new to studying the C language, for them it's important to understand even trivial things, for experts it's superfluous for sure...

File: src/.clang-format (hunk 1 above)
Comment (Sandroid75) [reply src/.clang-format:1]: sorry my mistake

File: src/.vscode/launch.json (hunk 2 above)
Comment (Sandroid75) [reply src/.vscode/launch.json:1]: sorry again

File: src/.vscode/tasks.json (hunk 3 above)
Comment (Sandroid75) [reply src/.vscode/tasks.json:1]: you are right....I forgot gitignore sorry

**Comment by jubalh** [comment]
If you have time you can do the proposed changes. And then force push to this branch for another review round :)

**Pull request closed without merging**